### PR TITLE
lastCardPlayed does not need to be a complex type

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/base/SpecialDesign.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export class SpecialDesign extends Card implements IProjectCard {
     });
   }
   public getRequirementBonus(player: Player): number {
-    if (player.lastCardPlayed !== undefined && player.lastCardPlayed.name === this.name) {
+    if (player.lastCardPlayed !== undefined && player.lastCardPlayed === this.name) {
```
Comment (bafolts) [inline src/cards/base/SpecialDesign.ts:27]: Similar to the change in `IndenturedWorkers` this conditional can be simplified to `if (player.lastCardPlayed === this.name)`.

File: src/cards/colonies/Conscription.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export class Conscription extends Card implements IProjectCard {
   }
 
   public getCardDiscount(player: Player) {
-    if (player.lastCardPlayed !== undefined && player.lastCardPlayed.name === this.name) {
+    if (player.lastCardPlayed !== undefined && player.lastCardPlayed === this.name) {
```
Comment (bafolts) [inline src/cards/colonies/Conscription.ts:32]: I would use the `if (player.lastCardPlayed === this.name)` for all of these checks.

**Review by bafolts — APPROVED** [review]
This is great change. Will lead to less source and should also be slightly more efficient.

File: src/cards/base/SpecialDesign.ts (hunk 1 above)
Comment (kberg) [reply src/cards/base/SpecialDesign.ts]: I fixed four of them.

**Pull request merged**